projects/1.21/assets/ars-elixirum-neoforge/elixirum/lang/zh_cn.json

```
@@ -0,0 +1,286 @@
+  "elixir.tier.grand": "卓越",
+  "elixir.tier.intense": "强效",
+  "elixir.tier.legendary": "传奇",
+  "elixir.tier.minor": "初级",
+  "elixir.tier.moderate": "中级",
+  "elixir.tier.pale": "淡弱",
+  "elixir.tier.supreme": "顶级",
+  "elixir.tier.weak": "弱效",
+  "elixirum.alchemy_properties.affix": " %s",
+  "elixirum.alchemy_properties.essence": " x%s %s",
+  "elixirum.alchemy_properties.title": "炼金性质：",
+  "elixirum.alchemy_properties.unknown": " ???",
+  "elixirum.compendium.affixes": "词缀",
+  "elixirum.compendium.affixes.note_1": "在发现原料的所有源质后，炼金师之眼即会在其提示文本中显示词缀。",
+  "elixirum.compendium.affixes.text_1": "词缀是原料的强力额外性质。词缀分为多种，有为所有源质提供百分比增益的，也有为某一种源质提供的，亦有为相邻原料提供的。影响相邻原料的词缀最常见，为灵药选定原料顺序时应当注意。",
+  "elixirum.compendium.all_essences": "源质类型总集",
+  "elixirum.compendium.contribution": "贡献",
+  "elixirum.compendium.creating_elixir": "制作灵药",
+  "elixirum.compendium.creating_elixir.step_1": "将玻璃炼药锅放在热源（营火、岩浆块等）上方。",
+  "elixirum.compendium.creating_elixir.step_2": "向炼药锅中注满水，等待其完全沸腾。",
+  "elixirum.compendium.creating_elixir.step_3": "依次加入原料。原料的加入顺序很重要，且会影响灵药的性质；但目前你还没有发现任何原料的性质，也就没法发挥出它们全部的潜能，随意加入即可。",
+  "elixirum.compendium.creating_elixir.step_4": "将灵药舀入空瓶，就可以使用了。如果运气不错，选择了一组不错的原料，成品可能会很实用；但成品也有可能剧毒。",
+  "elixirum.compendium.creating_elixir.step_5": "若仅需清空炼药锅，对其使用空铁桶即可。",
+  "elixirum.compendium.creating_elixir.text_1": "你的第一份灵药，需要的只是一台玻璃炼药锅、一桶水、独特原料若干、再加一个玻璃瓶而已。",
+  "elixirum.compendium.creating_elixir.text_2": "到此为止，你已经制成了第一份灵药，颜色大概是带点棕色，还有不少微弱和淡弱的效果。这只是炼金师的入门；要酿造出真正传奇般的灵药，探索原料性质、据此决定其顺序，这两步必不可少。向世界尽情宣告魔法炼金的真正力量吧！",
+  "elixirum.compendium.discovering_essences": "发现源质",
+  "elixirum.compendium.discovering_essences.note_1": "如在上述步骤后，原料的提示文本中列表的末尾依然显示“???”，即表示原料中还有未发现的源质，应当再次酿造。",
+  "elixirum.compendium.discovering_essences.step_1": "向酿造台中放入烈焰粉作为燃料。",
+  "elixirum.compendium.discovering_essences.step_2": "放入三个玻璃瓶，用于盛装源质。",
+  "elixirum.compendium.discovering_essences.step_3": "最后，放入原料，静待酿造过程完成即可。",
+  "elixirum.compendium.discovering_essences.text_1": "默认情况下，佩戴炼金师之眼时只能看到原料中是否存在某种源质，而无法得到完整的列表和权重。若要检验出原料的所有性质，你需要一台酿造台、用以研究的原料、烈焰粉，以及玻璃瓶若干。",
+  "elixirum.compendium.discovering_essences.text_2": "酿造结束后，原料会分解成各源质组分。若原料中有超过3种源质，则会在其中随机选取3种产出。取走酿造台中的源质瓶后，原料的提示文本便会显示所发现的源质与其权重，为原料的选取提供更多信息！",
+  "elixirum.compendium.essences_and_ingredients": "源质与原料",
+  "elixirum.compendium.essences_and_ingredients.note_1": "如需检验物品拥有何种源质，而又不想将其投入炼药锅，可以试试炼金师之眼。只要物品的提示文本中出现一段炼金性质的描述，那就可以用作炼金原料！",
+  "elixirum.compendium.essences_and_ingredients.text_1": "含有源质的物品均可用作灵药的原料。源质是效果的本质形态。物品中的每一种源质都有权重；成品灵药中效果的强度和持续时间，即由对应源质的权重总和决定。",
+  "elixirum.compendium.essences_and_ingredients.text_2": "灵药原料提示：\n - 单次灵药炼制过程中最多只能使用9种原料。\n - 没有必要加进多个同种原料！这么做什么都不会发生。",
+  "elixirum.compendium.introduction": "介绍",
+  "elixirum.compendium.introduction.note_1": "普通的药水依然会作为稀有战利品出现，但无法再以传统的方式合成。",
+  "elixirum.compendium.introduction.text_1": "Ars Elixirum将药水制备、药水应用、药水多样性拓展到了极致。历经探索与实验之后，你便能形成独属于你自己的灵药（药水的强力替代品）配方，还能保存再加修改。",
+  "elixirum.discovered_essences": "你已经发现了%3$s种原料中的%1$s种源质，总共有%2$s种源质。",

```

原文：You have discovered %s of %s essences, distributed among %s ingredients.

```suggestion
  "elixirum.discovered_essences": "在所有的%2$s种源质中，你已经从%3$s种原料中发现了%1$s种源质。",
```